myapp/spec/system/tasks_spec.rb

```diff
@@ -8,7 +8,7 @@
       visit tasks_path
 
       # 正しい情報が表示されていること
-      expect(page).to have_content 'Tasks'
+      expect(page).to have_content I18n.translate('tasks.index.title')
```

Comment:
It is better to use a fixed value for the expected value of the test.

- It's possible to see the specific process by looking at the test file only.
- It is preferable as a test case that the test fails because the behavior has changed when settings such as i18n are rewritten.